enterprise/ha/src/test/java/jmx/HaBeanIT.java

```diff
@@ -68,7 +71,22 @@
     public static final ClusterRule clusterRule = new ClusterRule( HaBeanIT.class )
             .withInstanceSetting( setting( "jmx.port", STRING, (String) null ), intBase( 9912 ) )
             .withInstanceSetting( HaSettings.ha_server, stringWithIntBase( ":", 1136 ) )
-            .withInstanceSetting( GraphDatabaseSettings.forced_kernel_id, stringWithIntBase( "kernel", 0 ) );
+            .withInstanceSetting( GraphDatabaseSettings.forced_kernel_id, stringWithIntBase( "kernel", 0 ) )
+            .withStoreDirInitializer( new ClusterManager.StoreDirInitializer()
```

Comment:
@martinfurmanski I think it should be default behavior, but I opted for the minimal fix since this is a patch release and all.

@digitalstain Yes, it makes sense for the ClusterRule to do some some cleaning by default. One issue however is that a directory is made once for each TestClass only, and not per TestMethod. So we would need to set a default StoreDirInitializer in the ClusterRule itself for that.

Should I make those changes?